tests/components/systemmonitor/test_repairs.py

```diff
@@ -94,7 +94,7 @@ async def test_migrate_process_sensor(
     assert resp.status == HTTPStatus.OK
     data = await resp.json()
 
-    assert data["type"] == FlowResultType.CREATE_ENTRY
+    assert data["type"] == FlowResultType.CREATE_ENTRY.value
```

Comment:
> idem

I think your original comment has disappeared - as I only see "idem" now but I don't see what it relates to.

Reply:
<img width="616" alt="image" src="https://github.com/home-assistant/core/assets/7083755/d0fbe482-d616-4583-b7dd-6045758cd789">

Reply:
In the case of `auth` and `systemmonitor`, the data is coming from json parsing, not from an internal class and we cannot use `is` statement.

I added `.value` to make this more obvious, but maybe it would be better to switch to standard strings instead.

Reply:
Better to use a string then I guess with a comment or someone else will probably "fix" it later.

Reply:
I think it makes sense to keep the enum value over a simple string.
Adding a comment is a nice suggestion.